hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/util/HivePartitionUtil.java

```diff
@@ -57,7 +57,7 @@ public static String getPartitionClauseForDrop(String partition, PartitionValueE
         // This is a decode operator for encode in KeyGenUtils#getRecordPartitionPath
         partitionValue = PartitionPathEncodeUtils.unescapePathName(partitionValue);
       }
-      partBuilder.add(config.getSplitStrings(META_SYNC_PARTITION_FIELDS).get(i) + "=" + partitionValue);
+      partBuilder.add(config.getSplitStrings(META_SYNC_PARTITION_FIELDS).get(i).toLowerCase() + "=" + partitionValue);
```

Comment:
We changed the partition key to lowercase when synchronizing hive.
https://github.com/apache/hudi/blob/d56072527b074e1cc7f5c34d8eba66540a14589d/hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java#L389-L405

Reply:
Is the regular table column case in-sensative?

Reply:
Does hive throws exception in this case, or does Hive support partition field as upper-case? Can we just fix the spark queries?

Reply:
Query is ok with lower case or uper case. 

This case happens if we drop upper case partition in spark sql.